scripts/run_example

```diff
@@ -2,18 +2,43 @@
 set -e
 set -x
 
-readonly NAME="${1}"
+IS_CPP=false
```

Comment:
I will create an issue to track this. I think you are right, it's a bit confusing right now.

Reply:
I agree with this too, perhaps only the hello world example will have implementations in both languages after all, so we probably don't need the `module/{rust,cpp}` extra nesting.

Reply:
I am not sure about this. I agree that the scripts shouldn't assume both of them are there and it is slightly convoluted the way we run them now, so maybe having a build/run script in each example folder would be the way to go. But I do think that for the purpose of "reference implementation" we should give users examples in all/most languages. A pattern I am thinking about: the SDK should support parity on Rust and C++ and we will need code to test this. Maybe not all examples should have both, but at least a few of them should.

Reply:
IMO parity should be tested by tests, not by examples. In practice we are using examples as tests currently, but really we should be writing proper tests that load some code and interact with the Node and expect something back.

Reply:
Reading back what I wrote in the previous comment - I really used the wrong word ("test") to express what I wanted, and it came across very differently from what I had in mind. Apologies for wasting your time. I totally agree with what you are saying and I wasn't (intentionally :P) trying to suggest otherwise.

Functionality shouldn't be tested by examples, we need proper testing for the SDK. However, the whole idea with having examples (if I understand it correctly), is to provide future developers samples of how to build Oak apps. It's not about testing, it's about showcasing, and this is why I think going for parity for "supported" languages is important, imho.